app/src/androidTest/java/org/mozilla/fenix/ui/robots/TabDrawerRobot.kt

```diff
@@ -52,6 +52,19 @@ import org.mozilla.fenix.helpers.matchers.BottomSheetBehaviorStateMatcher
  * Implementation of Robot Pattern for the home screen menu.
  */
 class TabDrawerRobot {
+
+    fun verifyTabUrl(url: String) {
+        val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
```

Comment:
This function name can mean so many things. Maybe, ```verifyBrowserTabsTrayURL()```

Reply:
Hi @AaronMT , done and thanks for the heads-up ! ☺️